config/options.go

```diff
@@ -482,6 +488,15 @@ func (o *Options) Validate() error {
 		log.Warn().Msg("config: cache url will be deprecated in v0.11.0")
 		o.DataBrokerURLString = o.CacheURLString
 	}
+	switch o.DataBrokerBackendStorageType {
+	case "memory":
+	case "redis":
```

Comment:
Thoughts on adding an exported const value to the memory and redis packages so we can refer to this as variables (e.g. memory.Name / redis.Name)

Reply:
👍